modules/test/playwright/tests/account-admin-web/accounts.spec.ts

```diff
@@ -191,3 +191,34 @@ test('LPD-18484 Add account website', async ({
 		page.getByRole('cell', {name: 'https://www.website.com'})
 	).toBeVisible();
 });
+
+test('LPD-25948 Can search for account group', async ({
+	accountAccountGroupsPage,
+	accountsPage,
+	apiHelpers,
+	page,
+}) => {
+	const account = await apiHelpers.headlessAdminUser.postAccount({
+		name: 'Account Name',
+		type: 'business',
+	});
+
+	const accountGroup = await apiHelpers.headlessAdminUser.postAccountGroup({
+		name: 'Account Group Name',
+	});
+
+	await apiHelpers.headlessAdminUser.assignAccountToAccountGroup(
+		account.externalReferenceCode,
+		accountGroup.externalReferenceCode
+	);
+
```

Comment:
let's this test to be more useful
Create a second accountGroup and assign it to the same user so when we do the search we will check that the first one is present but the second one not, otherwise with this test we don't know if the search works or not :)